src/components/dropdown/dropdown.stories.ts

```diff
@@ -191,8 +246,58 @@ export const darkThemeRTL_TestOnly = (): string => html`
     type="${select("type", ["click", "hover"], "click")}"
     ${boolean("disable-close-on-select", false)}
     ${boolean("disabled", false)}
+  >
+    <calcite-button slot="dropdown-trigger">Open Dropdown</calcite-button>
+    <calcite-dropdown-group
+      selection-mode="${select("group selection mode", ["single", "multi", "none", "multiple"], "single")}"
+      group-title="Icon Start"
+    >
+      <calcite-dropdown-item icon-start="list">List</calcite-dropdown-item>
+      <calcite-dropdown-item icon-start="grid" selected>Grid</calcite-dropdown-item>
+      <calcite-dropdown-item icon-start="table">Table</calcite-dropdown-item>
+    </calcite-dropdown-group>
+    <calcite-dropdown-group
+      selection-mode="${select("group selection mode", ["single", "multi", "none", "multiple"], "single")}"
+      group-title="Icon End"
+    >
+      <calcite-dropdown-item icon-end="list">List</calcite-dropdown-item>
+      <calcite-dropdown-item icon-end="grid" selected>Grid</calcite-dropdown-item>
+      <calcite-dropdown-item icon-end="table">Table</calcite-dropdown-item>
+    </calcite-dropdown-group>
+    <calcite-dropdown-group
+      selection-mode="${select("group selection mode", ["single", "multi", "none", "multiple"], "single")}"
+      group-title="Icon Both"
+    >
+      <calcite-dropdown-item icon-start="list" icon-end="data-check">List</calcite-dropdown-item>
+      <calcite-dropdown-item icon-start="grid" icon-end="data-check" selected>Grid</calcite-dropdown-item>
+      <calcite-dropdown-item icon-start="table" icon-end="data-check">Table</calcite-dropdown-item>
+    </calcite-dropdown-group>
+  </calcite-dropdown>
+`;
+
+WithIconsDarkTheme.storyName = "With Icons - Dark theme";
+WithIconsDarkTheme.parameters = { themes: themesDarkDefault };
+
+export const GroupsAndSelectionModesDarkTheme = (): string => html`
```

Comment:
Let's drop this one since it was removed in the [storybook cleanup PR](https://github.com/Esri/calcite-components/pull/5257/files#diff-bc2137321903a9338f046ec0516d2cd615e61bad7075e894432746133fe121a5).